docs/contributing.rst

```diff
@@ -79,10 +80,10 @@ tests the root test directory has to be specified using the ``--testpath`` comma
 line option, e.g. ``./build/test/soltest -- --testpath ./test``.
 
 To disable the z3 tests, use ``./build/test/soltest -- --no-smt --testpath ./test`` and
```

Comment:
It may be fine to ignore this, but I think this command will probably not work (it neither has ``--ipcpath``, nor ``--no-ipc``, so it'll fail). Maybe better to first document ``--no-ipc`` and then to add ``--no-smt`` on top of that, so that every command can actually be run.